ui/cap-react/src/antd/forms/templates/ArrayFieldTemplates/NormalArrayFieldTemplate.js

```diff
@@ -313,7 +313,7 @@ const NormalArrayFieldTemplate = ({
         <Col span={24} style={{ marginTop: "5px" }} className="nestedObject">
           <Row>
             {items && (
-              <Col classdName="row array-item-list" span={24}>
+              <Col className="row array-item-list" span={24}>
```

Comment:
maybe we should remove if there is no difference?

Reply:
We actually have many of these `className`s around which probably come from copypasting from the original rjsf classes and which we can remove little by little since they are not used.